tests/unit/components/HdLazyImage.spec.js

```diff
@@ -0,0 +1,25 @@
+import { mount } from '@vue/test-utils';
+import HdLazyImage from '@/components/HdLazyImage.vue';
+import ITEMS from '@/stories/mocks/GALLERY_ITEMS';
+
+describe('HdLazyImage', () => {
+  it('renders with the image as background', () => {
+    const wrapper = mount(HdLazyImage, {
```

Comment:
When you're passing props, you have to pass them under `propsData` key. 

Like so: 

```javascript
    const wrapper = mount(HdLazyImage, {
      propsData:{
        src: ITEMS[0].image,
        background: true,
      }
    });
```

Otherwise you're snapshots will render empty (see under comment by @viniciuskneves )